### ISSUE TITLE
Testable Swift Example Issue

### ISSUE THREAD

**Issue by imsrc21** [kif-framework/KIF#970]
Hey,

I ran one of example(Testable Swift) from downloaded KIF repo and found below error: 

**Assertions: The step timed out after 10.00 seconds: Failed to find accessibility element with the label "Red"
File: SimpleObjCTest.m:20**

I found that this issue is related to accessibility and tried to debug but couldn't find anything. 
I am using XCode 8.3.2 and device(or simulator) running iOS 10.3.

Thanks in advance.

**Comment by justinseanmartin** [comment]
Looks like the test itself is just broken. I'll fix it and add it to be run as part of Travis builds to make sure we don't regress it.
